extensions/wikia/AdEngine/js/lookup/services.js

```diff
@@ -14,17 +14,15 @@ define('ext.wikia.adEngine.lookup.services', [
 	require.optional('ext.wikia.adEngine.lookup.prebid'),
 	require.optional('ext.wikia.adEngine.lookup.amazonMatch'),
 	require.optional('ext.wikia.adEngine.lookup.openXBidder'),
-	require.optional('ext.wikia.adEngine.lookup.rubicon.rubiconFastlane'),
-	require.optional('ext.wikia.adEngine.lookup.rubicon.rubiconVulcan')
-], function (log, prebid, amazonMatch, oxBidder, rubiconFastlane, rubiconVulcan) {
+	require.optional('ext.wikia.adEngine.lookup.rubicon.rubiconFastlane')
+], function (log, prebid, amazonMatch, oxBidder, rubiconFastlane) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Function has 5 parameters which is greater than 4 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)